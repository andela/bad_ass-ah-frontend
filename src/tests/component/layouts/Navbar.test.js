import React from 'react';
import { shallow } from 'enzyme';
import Navbar from '../../../components/layouts/Navbar';

describe('Navbar Component', () => {
  test('renders without crashing', () => {
    const component = shallow(<Navbar />);
    expect(component).toMatchSnapshot();
  });
});
