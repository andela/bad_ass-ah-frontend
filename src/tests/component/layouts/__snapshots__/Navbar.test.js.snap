// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navbar Component renders without crashing 1`] = `ShallowWrapper {}`;
